projects/core/styles/theme/variables.less

```diff
@@ -1,12 +1,23 @@
 @font-fallback: -apple-system, 'BlinkMacSystemFont', system-ui, 'Roboto', 'Segoe UI', 'Helvetica Neue', sans-serif;
 
 :root {
-    // Deprecated
-    --tui-heading-font: 'Manrope', @font-fallback;
-    --tui-text-font: 'Manrope', @font-fallback;
     // Fonts
-    --tui-font-heading: var(--tui-heading-font);
-    --tui-font-text: var(--tui-text-font);
+    --tui-font-heading: 'Manrope', @font-fallback;
+    --tui-font-heading-1: bold 50px/56px var(--tui-font-heading);
+    --tui-font-heading-2: bold 40px/44px var(--tui-font-heading);
+    --tui-font-heading-3: bold 30px/36px var(--tui-font-heading);
+    --tui-font-heading-4: bold 28px/32px var(--tui-font-heading);
+    --tui-font-heading-5: bold 24px/28px var(--tui-font-heading);
+    --tui-font-heading-6: bold 20px/24px var(--tui-font-heading);
+    --tui-font-text: 'Manrope', @font-fallback;
+    --tui-font-text-xl: normal 19px/28px var(--tui-font-text);
+    --tui-font-text-l: normal 17px/24px var(--tui-font-text);
+    --tui-font-text-m: normal 15px/24px var(--tui-font-text);
+    --tui-font-text-s: normal 13px/20px var(--tui-font-text);
+    --tui-font-text-xs: normal 11px/16px var(--tui-font-text);
+    --tui-font-caption: 'Manrope', @font-fallback;
```

Comment:
I don't think we need captions, let's just drop them.

Reply:
Do you mean captions in general and set the style inline or just drop the font face definition of captions and use the normal text font?

Reply:
I don't think it is used anywhere in public code, other than on Typography page itself.

Reply:
They are in use in islands as category headlines. Should I remove the category styles and replace them with inline styles?

Reply:
Right. Yes, please do so.

Reply:
Done